game/src/contrib/systems/DebuggerSystem.java

```diff
@@ -140,12 +140,12 @@ public static void TELEPORT(Point targetLocation) {
      * load.
      */
     public static void TOGGLE_LEVEL_SIZE() {
-        switch (Game.LEVELSIZE) {
-            case SMALL -> Game.LEVELSIZE = LevelSize.MEDIUM;
-            case MEDIUM -> Game.LEVELSIZE = LevelSize.LARGE;
-            case LARGE -> Game.LEVELSIZE = LevelSize.SMALL;
+        switch (Game.levelSize()) {
+            case SMALL -> Game.levelSize(LevelSize.MEDIUM);
+            case MEDIUM -> Game.levelSize(LevelSize.LARGE);
+            case LARGE -> Game.levelSize(LevelSize.SMALL);
```

Comment:
Ehrlich gesagt verstehe ich den Schnipsel nicht.

Du fragst in Game die levelSize ab, um sie dann wieder zu setzen?

Reply:
Andererseits: das ist ja nur der Debugger. Feel free ignore + "resolve" this comment.